Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml

```diff
@@ -14,7 +14,9 @@
     <DockPanel x:Name="mainDockPanel"  HorizontalAlignment="Stretch" LastChildFill="true" Margin="0,0,0,0" VerticalAlignment="Stretch" Background="{StaticResource $BackgroundColor_LightGray}">
         <!-- Grid Title -->
         <Border x:Name="xSimpleHeader" DockPanel.Dock="Top" Visibility="{Binding  ElementName=lblTitle, Path=Visibility}" Background="{Binding ElementName=lblTitle, Path=Background}" BorderThickness="1" BorderBrush="{StaticResource $BackgroundColor_DarkBlue}"  >
-            <Label x:Name="xSimpleHeaderTitle" Style="{DynamicResource $ucGridTitleLightStyle}" Margin="0,1,0,1">Grid Title</Label>
+            <DockPanel>
```

Comment:
this change will impact all pages having Grids with title... you can use your own title in the mergingPage and hide the Grid title in case you have issues with placing it on center

Reply:
done ! Was mistakenly Pushed,